taskcluster/ci/config.yml

```diff
@@ -129,6 +129,12 @@ taskgraph:
             default-repository: https://github.com/mozilla-extensions/dynamic-probe-telemetry-extension
             default-ref: main
             type: git
+        quicksuggestweather:
+            name: "Firefox Quick Suggest Weather"
+            project-regex:  firefox-quick-suggest-weather$
+            default-repository: https://github.com/mozilla-extensions/firefox-quick-suggest-weather
```

Comment:
```suggestion
            default-repository: git@github.com:mozilla-extensions/firefox-quick-suggest-weather
```

Reply:
For private repos, we need the ssh format (`git@github.com:org/repo`). That's why the build task is hanging in the PR checks.